mne_bids/config.py

```diff
@@ -57,24 +57,78 @@
 
 # accepted BIDS formats, which may be subject to change
 # depending on the specification
-# XXX: Add support for MEG and EEG
 BIDS_IEEG_COORDINATE_FRAMES = ['acpc', 'pixels', 'other']
+BIDS_MEG_COORDINATE_FRAMES = ['ctf', 'elektaneuromag',
+                              '4dbti', 'kityokogawa',
+                              'chietiitab', 'other']
+BIDS_EEG_COORDINATE_FRAMES = ['captrak']  # XXX: We don't include BESA
```

Comment:
```suggestion
BIDS_EEG_COORDINATE_FRAMES = ['captrak']
```

BESA has been completely removed now as a bugfix, we don't even need to mention it